MediaBrowser.Model/Updates/InstallationInfo.cs

```diff
@@ -8,33 +8,21 @@ namespace MediaBrowser.Model.Updates
     public class InstallationInfo
     {
         /// <summary>
-        /// Gets or sets the id.
+        /// Gets or sets the guid.
         /// </summary>
-        /// <value>The id.</value>
-        public Guid Id { get; set; }
+        /// <value>The guid.</value>
+        public string Guid { get; set; }
```

Comment:
I'm not getting this change. Why rename a field, and why change its type?..

Reply:
I did this to match the other object in the VersionInfo class.

Reply:
We should make them all `PascalCase`